html/semantics/scripting-1/the-script-element/execution-timing/151.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Hmm, you might want to see https://github.com/web-platform-tests/wpt/blob/master/dom/nodes/insertion-removing-steps/Node-appendChild-script-in-script.tentative.html. It's related to https://github.com/whatwg/dom/pull/1261 and https://github.com/whatwg/html/pull/10188, and once the latter lands, I think we'll need a different set of expectations corresponding with the simpler spec.

In fact, I'm wondering if we don't even need the test in this PR once we make the existing one I linked to above, non-tentative. Thoughts?

Reply:
The one you linked to is basically the same as 128, right, in that it relies on the parser-inserted script s1 being unflagged as parser-inserted? So I think this variant still has some small but nonzero value compared to either of them.

Reply:
> So I think this variant still has some small but nonzero value compared to either of them.

Yep, that may well address the point about whether we should remove this test or not. But I still think it could be good to update the expectations to match what we're pursuing in HTML regarding script scheduling + children changed events.